arbiter-core/src/math.rs

```diff
@@ -55,20 +126,51 @@ mod tests {
     }
 }
 
-/// Converts a float to a WAD fixed point prepared U256 number.
+/// Converts a floating-point number to a WAD fixed-point representation using
+/// `U256`.
+///
+/// WADs are fixed-point numbers with (usually) 18 decimal places. They are
+/// useful for representing decimals in smart contracts.
+///
 /// # Arguments
-/// * `x` - Float to convert. (f64)
+///
+/// * `x` - The floating-point number to convert.
+///
 /// # Returns
-/// * `U256` - Converted U256 number.
+///
+/// Returns the `U256` representation of the WAD fixed-point number.
+///
+/// # Examples
+///
+/// ```
+/// # use arbiter_core::math::float_to_wad;
+/// let wad_val = float_to_wad(1.23);
+/// ```
 pub fn float_to_wad(x: f64) -> U256 {
     U256::from((x * 1e18) as u128)
 }
 
-/// Converts a float to a WAD fixed point prepared U256 number.
+/// Converts a WAD fixed-point number, represented as `U256`, back to a
+/// floating-point number.
+///
+/// WADs are fixed-point numbers with 18 decimal places.
+///
 /// # Arguments
-/// * `x` - WAD to convert. (U256)
+///
+/// * `x` - The `U256` representation of the WAD fixed-point number.
+///
 /// # Returns
-/// * `f64` - Converted f64 number.
+///
+/// Returns the floating-point representation of the number.
+///
+/// # Examples
+///
+/// ```
+/// # use [YOUR_CRATE]::math::{float_to_wad, wad_to_float};
```

Comment:
[YOUR_CRATE] seems to be leftover from chatgpt

Reply:
Good catch lol